docs/docusaurus/docs/cloud/setup/setup_cloud.md

```diff
@@ -0,0 +1,183 @@
+---
+sidebar_label: Try GX Cloud
+title: Try GX Cloud
+id: setup_cloud
+description: An end-to-end reference to help new users set up GX Cloud.
+---
+
+import TechnicalTag from '@site/docs/term_tags/_tag.mdx';
+import SetupAndInstallForSqlData from '/docs/components/setup/link_lists/_setup_and_install_for_sql_data.md'
+import SetupAndInstallForFilesystemData from '/docs/components/setup/link_lists/_setup_and_install_for_filesystem_data.md'
+import SetupAndInstallForHostedData from '/docs/components/setup/link_lists/_setup_and_install_for_hosted_data.md'
+import SetupAndInstallForCloudData from '/docs/components/setup/link_lists/_setup_and_install_for_cloud_data.md'
+
+If you're new to GX Cloud, the information provided here is intended to demonstrate GX Cloud features and functionality. You'll connect to a Data Source, build an Expectation using sample Batch data, validate data with the Expectation, and review Validation Results.
+
+After you've tested GX Cloud features and functionality, you can connect to your Data Source and create Expectations that are specific to your business needs.
```

Comment:
Our product is calling it Datasource

Reply:
@joshzzheng I am confirming the convention in GX Cloud. Usually, writers match the UI when creating content. In this case, a convention already exists with OSS (Data Source). I want to confirm plans before making the edit.

Reply:
Slack conversation [here](https://greatexpectationslabs.slack.com/archives/CBWF8GCDT/p1695146672193279). It appears we're now using _Datasource_.